admin/sql/create_indexes.sql

```diff
@@ -42,4 +42,6 @@ CREATE INDEX user_id_ndx_user_timeline_event ON user_timeline_event (user_id);
 CREATE INDEX event_type_ndx_user_timeline_event ON user_timeline_event (event_type);
 CREATE INDEX user_id_event_type_ndx_user_timeline_event ON user_timeline_event (user_id, event_type);
 
+CREATE UNIQUE INDEX user_id_rec_mbid_ndx_pinned_track ON pinned_track (user_id, recording_mbid);
```

Comment:
We're unlikely to want to fetch data from this table using both user_id and recording_mbid. The normal pattern I expect is to ask "what recordings does this user have pinned"? So, this index should be on user_id only.

Reply:
Also, this should not be a UNIQUE index, otherwise a user can never pin the same track twice.